### ISSUE TITLE
Issues on Vesktop

### ISSUE THREAD

**Issue by 1RB** [s-k-y-l-i/discord-themes#2]
# TLDR
- Acrylic Harmony Transparent doesn't make chat list transparent on Vesktop
- Call information/stats aren't rendered properly on smaller dimensions of the window
- Are there extra steps (other than micaforeveryone) to improve experience on Vesktop?

<hr/>

Love the theme ❤️, there's some issues with it working on vesktop, i think its on their side for the most part.

## 1.
- `Acrylic Harmony Transparent` theme won't work as well probably because vesktop uses its own title bar.

|What it should look like| What it actually looks like|
|----|----|
|![image](https://private-user-images.githubusercontent.com/7442913/309529732-cb7faaea-5cdd-4671-854b-cecdd743b160.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEwMTk5MjcsIm5iZiI6MTcxMTAxOTYyNywicGF0aCI6Ii83NDQyOTEzLzMwOTUyOTczMi1jYjdmYWFlYS01Y2RkLTQ2NzEtODU0Yi1jZWNkZDc0M2IxNjAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDMyMSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAzMjFUMTExMzQ3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmM1YTA1OWRlMGQzYTUzYWVhMDY4Zjk1ODQ2MDY4Y2ZkNzYyZjg3N2MxODU3YTJkYjY5YzkzNzAyZmExZWQ3MSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.vweeW-zvcHzRb4c764Kc3d15BYaQQB--NDpxxbDZPiA) |![image](https://github.com/s-k-y-l-i/discord-themes/assets/86501179/0424339d-748f-4873-a1e7-926b71e5208d)|

I've tried playing around with the vesktop settings, such as `Disable the Window Frame`, `Discord Titlebar`, but i haven't had much luck with it. Is there any specific settings (or anything else in general) that i should have in order to have a better experience with the theme?

I want to add that the `Acrylic Harmony` theme works better but i like the transparency on the titlebar and server list only option

![image](https://github.com/s-k-y-l-i/discord-themes/assets/86501179/2c8eb383-ee52-4535-b752-e019765c6ff1)

## 2.
- When using the `Smooth Harmony` theme, the call information is rendered weirdly in smaller dimensions

| | | | 
|--------|--------|--------|
| ![image](https://github.com/s-k-y-l-i/discord-themes/assets/86501179/03d8bab8-d21a-4a5a-981b-e820d245e31b) | ![image](https://github.com/s-k-y-l-i/discord-themes/assets/86501179/af33770a-bc14-4a1e-9e3a-e60ce433989b) | ![image](https://github.com/s-k-y-l-i/discord-themes/assets/86501179/c832aae8-4bd1-4c41-84c9-cdbdb1c9551b) | 

Sorry if everything is all over the place, feel free to ask for more clarification

**Comment by 1RB** [comment]
I've also noticed that when using the `Acrylic Harmony` theme it is hard to read threads opened in side-by-side view

![image](https://github.com/s-k-y-l-i/discord-themes/assets/86501179/999333c2-9f02-4de9-8e7c-52a4e5988d49)

**Comment by s-k-y-l-i** [comment]
Thanks for the feedback! I made a lot of updates today, but I will check on these. I mainly test only the Smooth-harmony theme and with the Acrylic theme on top, but i haven't tested the Acrylic as a standalone since about a week.
I'll fix these and report back when done!

**Comment by 1RB** [comment]
Yeah, I've been messing around w the different themes, mixing and matching etc and just came across these little bugs. Really love the themes though. Keep it up!

**Comment by s-k-y-l-i** [comment]
I think I've fixed these, can you try now?

**Comment by 1RB** [comment]
- [x] `Acrylic Harmony Transparent Frame` - Vesktop 
I got it to work, for some reason the `Discord Titlebar` setting under **Vesktop Settings** wasn't working properly, but after a few restarts it now renders the custom discord titlebar👍

- [ ] `Smooth Harmony` - Call Information

|| ❌ No Change | ✅ Fixed | ❌ No Change | 
|-------|--------|--------|--------|
|Before| ![image](https://github.com/s-k-y-l-i/discord-themes/assets/86501179/03d8bab8-d21a-4a5a-981b-e820d245e31b) | ![image](https://github.com/s-k-y-l-i/discord-themes/assets/86501179/af33770a-bc14-4a1e-9e3a-e60ce433989b) | ![image](https://github.com/s-k-y-l-i/discord-themes/assets/86501179/c832aae8-4bd1-4c41-84c9-cdbdb1c9551b) | 
|After|![image](https://github.com/s-k-y-l-i/discord-themes/assets/86501179/a161c87c-01c1-4f62-96eb-19ddf6166faf)|![image](https://github.com/s-k-y-l-i/discord-themes/assets/86501179/cb8ec96f-66c8-4a8b-8124-d4aeddfcf0ef)|![image](https://github.com/s-k-y-l-i/discord-themes/assets/86501179/b67ab718-a895-4719-b0f3-420f27b05a9f)|

- [x] `Acrylic Harmony` - threads 

![image](https://github.com/s-k-y-l-i/discord-themes/assets/86501179/0fdb84f7-eb1f-4cfa-bf4e-bb88800d81b3)
